### PR TITLE
UI/Selector Vue component

### PR DESCRIPTION
**Pull request by andrewdds**
This creates a jinja macro/vue component which is designed to replace the wtforms Select input.
It expects as a parameter a wtforms select field "object". Array of choices tuples can take a third value to supply a description.

role-picker

### CODE REVIEW

File: atst/filters.py [hunk 1]
```diff
@@ -46,6 +47,8 @@ def mixedContentToJson(value):
         value["task_order"] = value["task_order"].filename
     return app.jinja_env.filters["tojson"](value)
 
+def toJson(data):
+    return json.dumps(data)
```
Comment (patricksmithdds) [inline atst/filters.py]: There's a [`tojson` filter](http://jinja.pocoo.org/docs/2.10/templates/#tojson) built into Jinja. Does this do anything different?

File: atst/forms/data.py [hunk 2]
```diff
@@ -102,3 +102,11 @@
     ("3-6 months", "3-6 months"),
     ("Above 12 months", "Above 12 months"),
 ]
+
+WORKSPACE_ROLES = [
+    ("admin", "Admin", "Lorem ipsum dolor sit amet, consectetur adipiscing elit."),
+    ("billing_auditor", "Billing Auditor", "Nullam sit amet sem volutpat, congue est nec, euismod leo."),
+    ("developer", "Developer", "Aenean suscipit facilisis eros, vitae malesuada dui egestas in."),
+    ("owner", "Owner", "Nullam luctus a arcu non placerat."),
+    ("security_auditor", "Security Auditor", "In ut leo vitae libero convallis finibus nec ac massa. Aliquam bibendum nulla at eros dapibus tristique."),
```
Comment (patricksmithdds) [inline atst/forms/data.py]: We have copy for these descriptions ([see the description of this issue](https://www.pivotaltracker.com/story/show/158072111)) -- can we add them now? It seems those descriptions have newlines in them, so I'd be interested to see that they place nice with the new display component.

File: atst/filters.py (hunk 1 above)
Comment (andrewdds) [reply atst/filters.py]: nope TIL

File: atst/forms/data.py (hunk 2 above)
Comment (andrewdds) [reply atst/forms/data.py]: I'll give it a try

Comment (andrewdds) [reply atst/forms/data.py]: It does not appear to respect `\n` but I don't think its a big deal.

File: styles/elements/_tooltip.scss [hunk 3]
```diff
@@ -68,7 +68,7 @@
       background: $color;
       color: black;
       padding: 24px;
-      border-radius: 5px;
+      //border-radius: 5px;
```
Comment (montana-mil) [inline styles/elements/_tooltip.scss]: did you want to keep this here?

**Pull request merged**